src/screens/ComponentLibraryScreen.tsx

```diff
@@ -291,6 +297,126 @@ function ComponentLibraryScreen() {
       <Backdrop bgClass="bg-slate-700/50">
         <TripHeader />
       </Backdrop>
+      <Title>Form Header</Title>
+      <View className="flex flex-row items-center justify-between px-12">
+        <IconButton
+          size={IconSize.Small}
+          icon="ChevronLeftIcon"
+          colour={useColours()['ghost']}
```

Comment:
can we define a `const colours = useColours()` instead of initiating each time please?

Reply:
👍